src/app/routes/registration/request-claim/request-claim.component.html

```diff
@@ -26,28 +26,47 @@ <h2 class="font-weight-600 mb-1">Enrol to {{ orgAppDetails?.appName || orgAppDet
           <div class="col-lg-12">
             <mat-label class="ml-3 pl-1" [ngStyle]="txtColor">Enrol For</mat-label>
             <mat-form-field class="mt-2 md-enroll" appearance="outline" floatPlaceholder="never" [ngStyle]="txtColor">
-              <mat-select class="md-enroll" placeholder="Enrol For" formControlName="enrolFor" (selectionChange)="enrolForSelected($event)">
+              <mat-select class="md-enroll" placeholder="Enrol For" formControlName="enrolFor"
+                (selectionChange)="enrolForSelected($event)">
                 <mat-option class="md-enroll" [ngStyle]="listColor" [value]="EnrolForType.ME">Myself</mat-option>
                 <mat-option class="md-enroll" [ngStyle]="listColor" [value]="EnrolForType.ASSET">My Asset</mat-option>
               </mat-select>
             </mat-form-field>
           </div>
+
+          <!-- On-chain and Off-chain -->
+
+          <!-- <ng-container>
+            <mat-checkbox class="pl-3 md-enroll" [disabled]="true" [ngStyle]="txtboxColor"><span
```

Comment:
@jrhender can we request on-chain and off-chain on the same time?

Reply:
Yes